GVFS/GVFS.Common/Database/PlaceholderTable.cs

```diff
@@ -26,6 +27,11 @@ public static void CreateTable(IDbConnection connection)
             }
         }
 
+        public static string NormalizePath(string path)

```

Comment:
I think this method is only used in `RemoveStartingWith`, should it be private (or is there somewhere else we should call it from)?